Cython/Compiler/ModuleNode.py

```diff
@@ -1881,24 +1887,37 @@ def generate_traverse_function(self, scope, code, cclass_entry):
         if base_type:
             # want to call it explicitly if possible so inlining can be performed
             static_call = TypeSlots.get_base_slot_function(scope, tp_slot)
+            code.putln("{")
```

Comment:
Could you move this into the two cases that need it? It's needlessly turns a simple one-line expression into three in a common case. (same below)